telegram/bot.py

```diff
@@ -132,6 +106,27 @@ def __init__(self, token, base_url=None, base_file_url=None, request=None, priva
                                                                   password=private_key_password,
                                                                   backend=default_backend())
 
+    def _message(self, url, data, **kwargs):
+        if kwargs.get('reply_to_message_id'):
+            data['reply_to_message_id'] = kwargs.get('reply_to_message_id')
```

Comment:
give assignment operators now! :D
But as long as we can't use those. double `get`'s are not needed. if `kwargs.get` returns a value we can use `kwargs['item']` within the iff clause.
I have no idea if it has any change in speed or not :D

Reply:
Wouldn't the most pythonic way be 

```suggestion
        if 'reply_to_message_id' in kwargs:
            data['reply_to_message_id'] = kwargs['reply_to_message_id']
```